lib/taste_tester/ssh.rb

```diff
@@ -24,9 +24,8 @@ class SSH
 
     attr_reader :output, :status
 
-    def initialize(host, timeout = 5, tunnel = false)
+    def initialize(host, tunnel = false)
```

Comment:
this would break any user of `tunnel`, so I went searching around... and it turns out, it's not used. Not by this class, and not by any callers, we should just nuke it. AFAICT, this was designed to be used by tunnel.rb, but it doesn't use it, at all.

but since you would be breaking it, lets kill it in this PR